### PR TITLE
SONARIAC-1504 Remove current logic for LimitRange detection

### PR DESCRIPTION
**Pull request by petertrr**

### CODE REVIEW

File: iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/AbstractLimitsCheckTest.java [hunk 1]
```diff
@@ -62,20 +58,6 @@ void testGetFirstChildElement() {
...
-  }
-
-  @Test
-  void shouldNotReportLimitIssueWhenProjectHasLimitRange() {
-    var projectContext = ProjectContext.builder().setLimitRange(Trilean.TRUE).build();
-    var checkContext = checkContextAfterReportMissingLimit(projectContext);
-    verify(checkContext, never()).reportIssue((TextRange) any(), any());
-  }
-
   CheckContext checkContextAfterReportMissingLimit(ProjectContext projectContext) {
```
Comment (jonas-wielage-sonarsource) [inline iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/AbstractLimitsCheckTest.java:61]: This method and the static object at the beginning are not used anymore. TestAbstractLimitsCheck most likely as well

File: iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesSensorTest.java [hunk 2]
```diff
@@ -397,13 +396,6 @@ private void assertSecondaryLocation(Issue.Flow flow, int startLine, int startLi
     assertThat(issueLocation.textRange()).hasRange(startLine, startLineOffset, endLine, endLineOffset);
   }
 
-  private HelmProcessor mockHelmProcessor(Map<String, String> inputToOutput) {
```
Comment (jonas-wielage-sonarsource) [inline iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesSensorTest.java:400]: Was this not used anymore?

**Review by jonas-wielage-sonarsource — APPROVED** [review]
LGTM, please have a look at my comment.

**Comment by jonas-wielage-sonarsource** [comment]
Also you need to disable the `qa_win` for now in order to merge, as it's broken.

File: iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesSensorTest.java (hunk 2 above)
Comment (petertrr) [reply iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesSensorTest.java:400]: Yes, it was only used by the deleted test

**Pull request merged**